zebra/cli/l3routes_vty.c

```diff
@@ -136,7 +136,7 @@ port_find_ipaddress (const struct ovsrec_port *port_row, char * ip_entry,
               strncpy(ip6_addrcopy, port_row->ip6_address,
                       sizeof(ip6_addrcopy));
               ipv6_address = strtok(ip6_addrcopy, "/");
-              if (!strcmp (ipv6_address, ip_entry))
+              if (ip_addr_cmp_af (AF_INET6, ipv6_address, ip_entry))
```

Comment:
Well in case if we already know exactly which AF we use ip_addr_cmp_af is better. ip_addr_cmp is used to compare addresses when AF is unknown to the caller. Still, I can replace all calls to ip_addr_cmp_af with calls to ip_addr_cmp, but it will be a bit slower, though easier to understand. Or I can merge 2 functions into 1, like  
```
ip_addr_cmp (int af, char *addr_str1, char *addr_str2) {
    switch (af) {
        case AF_INET: ...
        case AF_INET6: ....
        default: af = ((strstr(addr_str1, ":") != NULL) ? AF_INET6 : AF_INET)
```